src/chainparams.h

```diff
@@ -56,7 +56,8 @@ class CChainParams
     const Consensus::Params& GetConsensus() const { return consensus; }
     const CMessageHeader::MessageStartChars& MessageStart() const { return pchMessageStart; }
     int GetDefaultPort() const { return nDefaultPort; }
-
+    /** This chain can have buried and bip9 soft fork parameters overridden with -bip9params and -buriedsfparams */
+    bool AllowsOverriddenSoftFork() const { return fAllowsOverriddenSoftFork; }
```

Comment:
I'm following the coding style of previous code in CChainParams which does not expose the fields.